pkg/cli/admin/release/extract.go

```diff
@@ -359,10 +361,19 @@ func (o *ExtractOptions) Run(ctx context.Context) error {
 		if o.Included {
 			context := "connected cluster"
 			inclusionConfig := manifestInclusionConfiguration{}
+
+			clientVersion, reportedVersion, VersionErr := version.ExtractVersion()
+			if VersionErr != nil {
+				return VersionErr
+			}
+			if reportedVersion == "" {
+				reportedVersion = clientVersion.String()
```

Comment:
Can `clientVersion.String()` be empty too? If yes then can we continue?

Reply:
I have to admit that I did not check it as it is reshuffled from the existing code.
But yes, it could be empty too.

https://github.com/openshift/oc/blob/4b2b3cc964839bd249a0a4d255f1cba7724e8869/vendor/k8s.io/apimachinery/pkg/version/types.go#L34-L37

and its value is from the build time:

https://github.com/openshift/oc/blob/267009391e55919093984b63c22d67cb2de0c700/pkg/version/version.go#L31-L33

We use it only in logging and it would not give us lots of troubles even if it is the empty string.